phalcon/mvc/model/resultset.zep

```diff
@@ -212,7 +212,7 @@ abstract class Resultset
 	 * Changes the internal pointer to a specific position in the resultset.
 	 * Set the new position if required, and then set this->_row
 	 */
-	public final function seek(int position) -> void
+	public final function seek(position) -> void
```

Comment:
why

Reply:
this one also looks fine on paper http://php.net/manual/en/class.seekableiterator.php will double check

Reply:
PHP Fatal error:  Declaration of Phalcon\Mvc\Model\Resultset::seek(int $position) must be compatible with SeekableIterator::seek($position) in Unknown on line 0

same as others, seems like internal interface is not properly set